plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/Messages.java

```diff
@@ -16,27 +16,22 @@
 
 package com.google.cloud.tools.eclipse.appengine.localserver;
 
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
 import org.eclipse.osgi.util.NLS;
 
 public class Messages extends NLS {
   private static final String BUNDLE_NAME =
       "com.google.cloud.tools.eclipse.appengine.localserver.messages"; //$NON-NLS-1$
 
-  public static String NOT_FACETED_PROJECT;
-  public static String GAE_STANDARD_FACET_MISSING;
-  public static String NEW_SERVER_DIALOG_PORT;
-  public static String NEW_SERVER_DIALOG_INVALID_PORT_VALUE;
-  public static String PORT_IN_USE;
-  public static String PORT_OUT_OF_RANGE;
+  private static final ResourceBundle RESOURCE_BUNDLE = ResourceBundle.getBundle(BUNDLE_NAME);
 
-  public static String CREATE_APP_ENGINE_RUNTIME_WIZARD_DESCRIPTION;
-  public static String CREATE_APP_ENGINE_RUNTIME_WIZARD_TITLE;
-  public static String OPEN_CLOUD_SDK_PREFERENCE_BUTTON;
-  public static String RUNTIME_WIZARD_CLOUD_SDK_NOT_FOUND;
-
-  static {
-    // initialize resource bundle
-    NLS.initializeMessages(BUNDLE_NAME, Messages.class);
+  public static String getString(String key) {
```

Comment:
You need a multi-arg version of this to handle PORT_IN_UDE and other messages that take arguments. see MessagesTest in one of the other packages for a way to test these.